paper-menu.html

```diff
@@ -27,34 +27,39 @@
       background: var(--paper-menu-background-color, --primary-background-color);
       color: var(--paper-menu-color, --primary-text-color);
 
-      mixin(--paper-menu);
+      @apply(--paper-menu);
     }
 
     :host > ::content > [disabled] {
       color: var(--paper-menu-disabled-color, --disabled-text-color);
     }
 
-    :host > ::content > .iron-selected {
+    :host > ::content > *:focus {
       position: relative;
+      outline: 0;
     }
 
-    :host > ::content > .iron-selected::after {
+    :host > ::content > *:focus:after {
       @apply(--layout-fit);
-
       background: currentColor;
       /* FIXME move to paper-styles for next widget */
```

Comment:
But not a big deal if nobody else is currently depending on `opacity:0.12;` and `opacity:0.26;`